rpm/README.md

```diff
@@ -0,0 +1,100 @@
+#  AWS Lambda YUM repository manager for S3
+
+Automatic YUM repository building inside S3 bucket using with lambda support
+
+## Setting up S3 and Lambda
+
+Clone the repo and get all other required files
+```
+git clone https://github.com/tactycal/lambdaRepos.git
+cd lambdaRepos/rpm
+pip3 install -t . -r requirements.txt
```

Comment:
Dependencies are not stated here. It's always a good idea for the first second level title to be **Requirements** section.